appsignals-tests/images/jdbc/src/main/java/software/amazon/opentelemetry/AppController.java

```diff
@@ -46,18 +46,25 @@ public void prepareDB() {
     logger.info("Application Ready");
   }
 
-  @GetMapping("/success")
+  @GetMapping("/success/CREATE DATABASE")
   @ResponseBody
-  public ResponseEntity<String> success() {
+  public ResponseEntity<String> successCreateDatabase() {
+    jdbcTemplate.execute("create database testdb2");
+    return ResponseEntity.ok().body("success");
```

Comment:
Yes indeed and you can get the next view:

```
JdbcPostgresTest > testSuccessCreateDatabase() FAILED
    org.opentest4j.AssertionFailedError:
    Expecting value to be true but was false
        at jdk.internal.reflect.GeneratedConstructorAccessor25.newInstance(Unknown Source)
        at java.base@17.0.11/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base@17.0.11/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at app//software.amazon.opentelemetry.appsignals.test.jdbc.JdbcContractTestBase.assertSuccess(JdbcContractTestBase.java:327)
        at app//software.amazon.opentelemetry.appsignals.test.jdbc.JdbcPostgresTest.testSuccessCreateDatabase(JdbcPostgresTest.java:53)
```

Reply:
Hm, that isn't a very helpful message. Do the sample application logs contain anything more helpful?

Reply:
this is the same exact result of other tests if they fail.

For example on the select success case other than having the server throw the exception like what I'm doing here we are manually checking that we are returning 1 as count as result of the query

https://github.com/BlueLort/aws-otel-java-instrumentation/blob/omarnsr-create-database-tests/appsignals-tests/images/jdbc/src/main/java/software/amazon/opentelemetry/AppController.java#L60

You would see a result in request body based on exception

https://github.com/BlueLort/aws-otel-java-instrumentation/blob/omarnsr-create-database-tests/appsignals-tests/images/jdbc/src/main/java/software/amazon/opentelemetry/AppController.java#L79

```
 public ResponseEntity handleGenericException(Exception ex) {
    if (ex instanceof HttpServerErrorException.InternalServerError) {
      return ResponseEntity.internalServerError().build();
    } else if (ex instanceof HttpClientErrorException.BadRequest) {
      return ResponseEntity.badRequest().build();
    } else if (ex instanceof BadSqlGrammarException) {
      return ResponseEntity.internalServerError().body("fault");
    }
    return ResponseEntity.ok().build();
  }
```
However it won't show right away, there is nothing logging that, to be able to log it you will need to check the failing test and go in depth 

as BadSqlGrammar would be the same as InternalServerError (where you try to create an existing db for example) 

you would only get that the response from server is NOT_OK and need to print body or trace to go further.

Reply:
Had a sync offline with @thpierce , we both agree it's hard to debug, but since it's around in the package, its a bigger scope of what is needed out of this PR

we will work on a fix in the future.